validator/slashing-protection/local/standard-protection-format/import.go

```diff
@@ -267,34 +267,34 @@ func saveHighestSourceTargetToDB(ctx context.Context, validatorDB db.Database, s
 			if err != nil {
 				return fmt.Errorf("%d is not a valid target: %v", target, err)
 			}
-			se, ok := validatorHighestSourceEpoch[pubKey]
+			se, ok := validatorLowestSourceEpoch[pubKey]
```

Comment:
I would rename these for clarity to better understand them in a separate PR. something like incomingAttSource and existingAttSource or something like that. no need to do in this one